src/test/python/complete_winml_tutorial.py

```diff
@@ -62,44 +60,24 @@ def bind_model(model, image_frame):
 @timed_op
 def evaluate_model(session, binding):
     results = session.evaluate(binding, "RunId")
-    o = results.outputs.lookup("softmaxout_1")
+    o = results.outputs["softmaxout_1"]
     result_tensor = winml.TensorFloat._from(o)
     return result_tensor.get_as_vector_view()
 
 def load_labels(labels_path):
-    import csv
-    labels = dict()
-    with open(os.fspath(labels_path)) as labels_file:
-        labels_reader = csv.reader(labels_file)
-        for label in labels_reader:
-            label_text = ', '.join(label[1:])
-            labels[int(label[0])] = ', '.join(label[1:])
-    return labels
+    with open(labels_path) as labels_file:
+        return {int(index): ', '.join(labels)
+                for index, *labels
+                in csv.reader(labels_file)}
 
 def print_results(results, labels):
-    topProbabilities = [0.0 for x in range(3)]
-    topProbabilityLabelIndexes = [0 for x in range(3)]
-
-    for i, result in enumerate(results):
-        for j in range(3):
-            if result > topProbabilities[j]:
-                topProbabilityLabelIndexes[j] = i
-                topProbabilities[j] = result
-                break
-
-    print()
-    for i in range(3):
-        print(labels[topProbabilityLabelIndexes[i]], "with confidence of", topProbabilities[i])
+    for confidence, label in sorted(zip(results, labels), reverse=True)[:3]:
+        print(f"{labels[label]} ({confidence * 100:.1f}%)")
 
 async def async_main():
-
-    from pathlib import Path
     cur_path = Path.cwd()
 
-    while (cur_path / "samples").is_dir() == False:
-        cur_path = cur_path.parent
-
-    winml_content_path = cur_path / "samples/python/winml_tutorial/winml_content"
+    winml_content_path = cur_path 
```

Comment:
this looks wrong. This code has to find the tutorial content from else where in the tree